graphics/nanovg/Portfile

```diff
@@ -11,10 +11,11 @@ maintainers         @jasonliu-- openmaintainer
 
 if {$subport eq "nanovg"} {
 
-github.setup        memononen $subport 2bead03bea43b2418060aaa154f972829995e663
-version             20200320
-revision            0
-
+set shorthash       bf2320d
+set fullhash        bf2320d1175122374a9b806d91e9e666c9336375
+set lastcommit      20190524
```

Comment:
you are going backwards here in version number, from a 2020 commit to a 2019 one. MacPorts will not consider this an 'update' unless you bump the epoch. So please add

```
epoch 1
```

However, first see my other comment about the fork being used.